src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable {
+    private double area;
```

Comment:
Storing `area` as a field might be redundant since the area can be calculated using the `calculateArea()` method. Consider removing the `area` field and its getter and setter methods to simplify the class design.